src/ifp/src/InitFloorplan.cc

```diff
@@ -502,14 +506,42 @@ void InitFloorplan::makeTracks(odb::dbTechLayer* layer,
         layer->getName());
     return;
   }
+
+  int layer_min_width = layer->getMinWidth();
+
   auto x_track_count = int((die_area.dx() - x_offset) / x_pitch) + 1;
-  grid->addGridPatternX(die_area.xMin() + x_offset, x_track_count, x_pitch);
+  int origin_x = die_area.xMin() + x_offset;
+  // Check if track origin is usable during router
+  if (layer_min_width / 2 > origin_x - die_area.xMin()) {
```

Comment:
>=
it can touch the edge, just not go beyond.  Similarly in the other checks

Reply:
I think my comments are not clear about these checks. If the if statement is true, I'm ignoring the first (or the last) x/y coordinate, because the wire would go out of the die area. If I use `>=`, I'll be removing the tracks when the wire is just touching the edge.

Perhaps a better comment is "Check if the track is **not** usable during routing".

Reply:
The code is correct but the comment is confusing.  Also I would have written
```
 if (origin_x - layer_min_width / 2 <  die_area.xMin()) {
```
its the same mathematically but in mind it is more natural to compare the two x coordinates than two widths.